### PR TITLE
User docs

### PR DESCRIPTION
**Pull request by Pickysaurus**
Added FAQ

### CODE REVIEW

File: docs/users/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Depending on your choice of browser, there may be additional considerations when troubleshooting this issue. 
+
+??? info ":fontawesome-brands-firefox: Setting the default protcol in Firefox"
+    Mozilla browsers (such as Firefox) include their own protocol handler settings, which override those set by your operating system. 
+
+    To access these settings click the menu button to the right of the address bar and select "Settings". Under the "General" category locate the section entitled "Files and Applications". 
+
+    ![The settinggs page in Mozilla Firefox 127.0.1 showing the Files and Applications section](./images/FirefoxProtocols.webp)
+
+    Here you will see the `nxm` protocol and the default app which will be used to open it. Using the drop-down you can select the default application for Firefox to use. Setting it to whichever option is set as "Default" will use the application specified by your operating system.
```
Comment (erri120) [inline docs/users/FAQ.md:35]: Don't know if it's worth mentioning, but the `nxm` entry will only appear if you've tried to open an `nxm` link before.

**Review by erri120 — APPROVED** [review]

File: docs/users/FAQ.md (hunk 1 above)
Comment (Pickysaurus) [reply docs/users/FAQ.md:35]: I think it's fine not to mention that. If you're on this page, you've tried to open an NXM link and it didn't work.

**Comment by Sewer56** [comment]
Note: The linked roadmap is not publicly visible

File: docs/users/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+This page includes answers to the commonly asked questions about the app. 
+
+## Why doesn't the app support my game?
+The Nexus Mods app is still in development and will not support specific games because these features have not been included yet. Our primary focus is to establish core functionality and foundational elements. 
+
+Adding support for a wide range of games it not a practical decision at this stage of developement, as it can introduce additional complexity and obscure underlying issues that need to be resolved first. The goal is to ensure the mod manager works seamlessly for a select few games before expanding compatibility in subsequent beta and final releases.
+
+Our plan is to go game-by-game and work with the modding community at each step to ensure we're building something great. To see which games are supported, coming next and planned for the future please see the [roadmap](https://trello.com/b/gPzMuIr3/nexus-mods-app-roadmap).
```
Comment (Sewer56) [inline docs/users/FAQ.md:8]: Note: The linked roadmap is not publicly visible

Comment (Pickysaurus) [reply docs/users/FAQ.md:8]: Yep, I am aware of this. It's up to Luke to finish it up and publish it.

Comment (erri120) [reply docs/users/FAQ.md:8]: I didn't even know we had one...

File: docs/users/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Browsers
+Depending on your choice of browser, there may be additional considerations when troubleshooting this issue. 
+
+??? info ":fontawesome-brands-firefox: Setting the default protcol in Firefox"
+    Mozilla browsers (such as Firefox) include their own protocol handler settings, which override those set by your operating system. 
+
+    To access these settings click the menu button to the right of the address bar and select "Settings". Under the "General" category locate the section entitled "Files and Applications". 
+
+    ![The settinggs page in Mozilla Firefox 127.0.1 showing the Files and Applications section](./images/FirefoxProtocols.webp)
```
Comment (Sewer56) [inline docs/users/FAQ.md:33]: Typo here 👉 `settinggs`

Comment (Pickysaurus) [reply docs/users/FAQ.md]: Fixed, thanks

**Review by Sewer56 — APPROVED** [review]

**Pull request merged**